installer/Util/V1/SchemaHelper.php

```diff
@@ -41,11 +45,22 @@ public function __construct(AbstractSchemaManager $schemaManager)
     }
 
     /**
-     * @return AbstractSchemaManager
+     * @return Connection|null
      */
-    public function getSchemaManager(): AbstractSchemaManager
+    private function getConnection(): ?Connection
     {
-        return $this->schemaManager;
+        return $this->getEntityManager()->getConnection()->getWrappedConnection();
+    }
```

Comment:
this is wrong. Don't use `EntityManagerHelperTrait` in installer. need to discuss this